Models/Games/BracketGame.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using CribblyBackend.Interfaces;
+
+namespace CribblyBackend.Models
+{
+    public class BracketGame : IGame
+    {
+        public List<Team> Teams { get; set; }
+        //Specify if game is a PlayIn game or Bracket game
+        public string Type { get; set; }
```

Comment:
Maybe we can define `Type` as an enum instead of a string

Reply:
Ahh, I actually meant to remove that property. We could get the type of game it is by simply calling the GetType method, so I figured we might not need it